affirm_example/__init__.py

```diff
@@ -169,22 +197,25 @@ def capture_transaction(transaction_id, amount=None):
             "amount": amount,
         }),
         headers={"Content-Type": "application/json"},
-        auth=(app.config["AFFIRM"]["PUBLIC_API_KEY"],
-              app.config["AFFIRM"]["SECRET_API_KEY"]),
+        auth=(public_api_key,
+              _get_secret_api_key(public_api_key)),
         **_get_extra_request_args()).json()
 
+
 def display_transaction_actions(template_data):
     kwargs = {}
     if app.config['USE_HTTPS']:
         kwargs.update({'_external': True, '_scheme': 'https'})
     for transaction_action in {"read", "update", "capture", "refund", "void"}:
         template_data["{0}_url".format(transaction_action)] = url_for(".transaction_admin_do",
-                                                                 transaction_action=transaction_action,
-                                                                 transaction_id=template_data["transaction_id"],
-                                                                 **kwargs)
+                                                                      transaction_action=transaction_action,
+                                                                      transaction_id=template_data["transaction_id"],
+                                                                      public_api_key=template_data["public_api_key"]
```

Comment:
I'm thinking should we make this a url params instead of inside the url path?

Reply:
sure - [1c29e5b](https://github.com/Affirm/affirm-example-python/pull/32/commits/1c29e5b7fdd85b9f1ba24882f3bcc737a60cb899)